frontend/src/features/mapSideBar/SidebarContents/subforms/LandUsageForm.tsx

```diff
@@ -54,6 +66,15 @@ export const LandUsageForm = <T extends any>(props: ILandUsageProps & FormikProp
           field={withNameSpace('zoningPotential')}
         />
       </Row>
+      <GenericModal
+        title="Land Classification Changed"
+        message="Remember to update any buildings to this new classification as well."
+        display={showSurplusActiveWarning}
+        setDisplay={setShowSurplusActiveWarning}
+        okButtonText="Ok"
+        handleOk={() => setShowSurplusActiveWarning(false)}
+        handleCancel={() => setShowSurplusActiveWarning(false)}
```

Comment:
Does the modal need an "Ok" and a "cancel" button?

Reply:
It is just informational, should just be a `Close` button.

Reply:
it doesn't have a cancel button, but the handleCancel function is used by the x icon.

Reply:
it is also used when the user clicks in any space outside the modal.